api/src/test/java/org/pmiops/workbench/api/OfflineUserControllerTest.java

```diff
@@ -156,4 +161,25 @@ private AuditProjectAccessRequest cloudTaskToAuditProjectAccessRequest(Task t) {
         .fromJson(
             t.getAppEngineHttpRequest().getBody().toStringUtf8(), AuditProjectAccessRequest.class);
   }
+
+  @Test
+  public void testCheckInitialCreditsExpiration() {
+    offlineUserController.checkInitialCreditsExpiration();
+
+    // Batch size is 2, so we expect 2 groups.
+    List<List<Long>> expectedRequests =
+        ImmutableList.of(ImmutableList.of(1L, 2L), ImmutableList.of(3L, 4L));
```

Comment:
newer Java versions have the native `List.of()` which works in the same way as the older Google-library `ImmutableList.of()`